back-end/controllers/snacksController.js

```diff
@@ -0,0 +1,99 @@
+/** @format */
+
+const express = require('express');
+const snacks = express.Router();
+
+const {
+	createSnack,
+	getAllSnacks,
+	getSnack,
+	updateSnack,
+	deleteSnack,
+} = require('../queries/snacks.js');
+
+const {
+	checkAddedSugar,
+	checkFiber,
+	checkName,
+	checkProtein,
+} = require('../validations/checkSnacks.js');
+
+const confirmHealth = require('../confirmHealth');
+const validName = require('../validations/validName');
+
+// INDEX ROUTE
+snacks.get('/', async (req, res) => {
+	const allSnacks = await getAllSnacks();
+	if (allSnacks[0]) {
+		res.status(200).json(allSnacks);
+	} else {
+		res.status(500).json({ error: 'server error!' });
+	}
+});
+
+// SHOW ROUTE
+snacks.get('/:id', async (req, res) => {
+	const { id } = req.params;
+	const snack = await getSnack(id);
+	if (snack.id) {
+		res.json(snack);
+	} else {
+		res.status(404).json({ error: 'Not found' });
+	}
+});
+
+// CREATE ROUTE
+snacks.post(
+	'/',
```

Comment:
lets leave the path on the same line as the .post

Reply:
I don't know how this happened because I normally do leave it on the same line. I will double check the code.